tests/memberclient.py

```diff
@@ -166,7 +165,8 @@ def run(args):
 
         LOG.debug("Further withdraw proposals fail")
         result = network.consortium.withdraw(3, primary, proposal_id)
-        assert result["error"]["code"] == params_error
+        assert result.error is not None
```

Comment:
Strange to assert result.error is not None but result.result. I think truthiness is fine in all cases, since we check specific keys immediately after. In fact it's not all that clear what that buys us because we should get a distinguishable exception if result.error really is None.

Reply:
Fair - I've removed these lines for consistency with the other assertions.